services/app/src/components/views/QuestHistory.tsx

```diff
@@ -1,55 +1,22 @@
 import * as React from 'react';
-import {QuestDetails} from '../../reducers/QuestTypes';
-import {SelectionListPhase, UserQuestInstance, UserQuestsType} from '../../reducers/StateTypes';
+import {UserQuestInstance, UserQuestsType} from '../../reducers/StateTypes';
 import Button from '../base/Button';
 import Card from '../base/Card';
 
 const Moment = require('moment');
 
 export interface QuestHistoryStateProps {
-  phase: SelectionListPhase;
   played: UserQuestsType;
-  selected: UserQuestInstance|null;
 }
 
 export interface QuestHistoryDispatchProps {
   onSelect: (selected: UserQuestInstance) => void;
-  onPlay: (details: QuestDetails) => void;
   onReturn: () => any;
 }
 
 export interface QuestHistoryProps extends QuestHistoryStateProps, QuestHistoryDispatchProps {}
 
-function renderDetails(props: QuestHistoryProps): JSX.Element {
-  const selected = props.selected;
-  if (!selected) {
-    return <Card title="Quest Details">Loading...</Card>;
-  }
-  const quest = selected.details;
-  const expansions = (quest.expansionhorror) ? <span><img className="inline_icon" src="images/horror_small.svg"/>The Horror</span> : 'None';
-  return (
-    <Card title="Quest Details">
-      <div className="searchDetails">
-        <h2>{quest.title}</h2>
-        <div>{quest.summary}</div>
-        <div className="author">by {quest.author}</div>
-        <div className="summary">Last played {Moment(selected.lastPlayed).fromNow()}</div>
-      </div>
-      <Button className="bigbutton" onClick={(e) => props.onPlay(quest)} id="play">Play</Button>
-      <Button onClick={(e) => props.onReturn()} id="back">Back</Button>
-      <div className="searchDetailsExtended">
-        <h3>Details</h3>
-        <div><strong>Expansions required: </strong>{expansions}</div>
-        <div><strong>Content rating:</strong> {quest.contentrating}</div>
-        <div><strong>Players:</strong> {quest.minplayers}-{quest.maxplayers}</div>
-        <div><strong>Genre:</strong> {quest.genre}</div>
-        <div><strong>Last updated: </strong> {Moment(quest.published).format('MMMM D, YYYY')}</div>
-      </div>
-    </Card>
-  );
-}
-
-function renderList(props: QuestHistoryProps): JSX.Element {
+const QuestHistory = (props: QuestHistoryProps): JSX.Element => {
   if (Object.keys(props.played).length === 0) {
     return (
       <Card title="Quest History">
```

Comment:
Question re: Quest History: What happens if you have multiple saves of the same quest ID? Want to make sure that flow works :) (that you're able to play both saves, that both saves correctly show their saved time, etc)

Reply:
It correctly propagates the save timestamp through to the quest preview and plays the quest save specified by the user.

FYI I'm working on a derivative PR now that actually turns the "Saved Quests" list into a "Saved & Offline Quests" list, where each item is a distinct quest that shows the number of saves (and if it's playable from the beginning while offline). Clicking through will go to the consolidated preview page and will list the saved instances of that quest as part of the preview... which means we'll also get this for free on e.g. previews from the search or featured quests pages.